src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          cletype="text"
+          class="distance"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+        ><br>
+
+        <label for="name">Name:</label>
+        <input
+          type="text"
+          class="distance"
+          id="name"
+          name="name"
+          autocomplete="off"
+        ><br>
+
+        <label for="age">How old are You?</label>
+        <input
+          type="number"
+          class="distance"
+          id="age"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+        ><br>
+
+        <label for="birth">Full date of birth:</label>
+        <input
+          type="date"
+          class="distance"
+          id="birth"
+          name="birth"
+        ><br>
+
+        <label for="checkbox">I accept the term of the agreement</label>
+        <input
+          type="checkbox"
+          id="checkbox"
+          name="checkbox"
+        >
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="email">E-mail:</label>
+        <input
+          type="email"
+          class="distance"
+          id="email"
+          name="email"
+          placeholder="email@example.com"
+        ><br>
+
+        <label for="password">Password:</label>
+        <input
+          type="password"
+          id="password"
+          name="password"
+        >
+
+      </fieldset>
```

Comment:
Don't add spaces between parent and child elements
```suggestion
        >
      </fieldset>
```